src/components/ReportPage/index.jsx

```diff
@@ -33,15 +35,18 @@ class ReportPage extends PureComponent {
       modalContent: {},
       type: '',
       isLoadingReports: false,
+      pagination: {
+        numberOfPages: 10,
```

Comment:
set the default 'numberOfPages' variable to 1 so that 10 would not appear when nothing is on the page

Reply:
Thank you for the feedback, I have made the necessary changes